easybuild/framework/easyconfig/default.py

```diff
@@ -115,6 +115,7 @@
     'patches': [[], "List of patches to apply", BUILD],
     'prebuildopts': ['', 'Extra options pre-passed to build command.', BUILD],
     'preconfigopts': ['', 'Extra options pre-passed to configure.', BUILD],
+    'prefer_ebpythonprefixes': [True, "Prefer EBPYTHONPREFIXES over PYTHONPATH when possible.", BUILD],
```

Comment:
This option should have a different name than its counterpart in `tools.options` because it is set in the oposite way. Following the discussion with @Micket , it should be called `--force-<other_option>`, as it can override that option.